### ISSUE TITLE
From first image to the second

### ISSUE THREAD

**Issue by yitong91** [sniklaus/pytorch-spynet#15]
Hi, this code is really helpful. I'm wondering how can I predict the second image from the first one + flow. 

I tried 

second_image_pred = backwarp(first_image, flow). But the result has strange color shift and the motion is not modeled well. 

Looking forward to your reply. Thanks!

**Comment by sniklaus** [comment]
I am afraid that, given the estimated flow from the first to the second image, you would have to use forward and not backward warping to reconstruct the second image from the first one. For more details, see: https://github.com/sniklaus/softmax-splatting
